tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -1187,42 +1187,23 @@ public function checkDownloadedContentMatches(
 	}
 
 	/**
-	 * @Then /^if the HTTP status code was "([^"]*)" then the downloaded content for multipart byterange should be:$/
+	 * @Then /^the downloaded content for multipart byterange should be:$/
 	 *
-	 * @param int $statusCode
 	 * @param PyStringNode $content
 	 *
 	 * @return void
 	 *
 	 */
-	public function theDownloadedContentForMultipartByterangeShouldBe(int $statusCode, PyStringNode $content):void {
-		$actualStatusCode = $this->response->getStatusCode();
-		if ($actualStatusCode === $statusCode) {
-			$actualContent = (string) $this->response->getBody();
-			$pattern = ["/--\w*/", "/\s*/m"];
-			$actualContent = \preg_replace($pattern, "", $actualContent);
-			$content = \preg_replace("/\s*/m", '', $content->getRaw());
-			Assert::assertEquals(
-				$content,
-				$actualContent,
-				"The downloaded content was expected to be '$content', but actually is '$actualContent'. HTTP status was $actualStatusCode"
-			);
-		}
-	}
-
-	/**
-	 * @Then /^if the HTTP status code was "([^"]*)" then the downloaded content should be "([^"]*)"$/
-	 *
-	 * @param int $statusCode
-	 * @param string $content
-	 *
-	 * @return void
-	 */
-	public function checkStatusCodeForDownloadedContentShouldBe(int $statusCode, string $content):void {
-		$actualStatusCode = $this->response->getStatusCode();
-		if ($actualStatusCode === $statusCode) {
-			$this->downloadedContentShouldBe($content);
-		}
+	public function theDownloadedContentForMultipartByterangeShouldBe(PyStringNode $content):void {
+		$actualContent = (string) $this->response->getBody();
+		$pattern = ["/--\w*/", "/\s*/m"];
+		$actualContent = \preg_replace($pattern, "", $actualContent);
+		$content = \preg_replace("/\s*/m", '', $content->getRaw());
+		Assert::assertEquals(
+			$content,
+			$actualContent,
+			"The downloaded content was expected to be '$content', but actually is '$actualContent'. HTTP status was $actualStatusCode"
```

Comment:
```suggestion
			"The downloaded content was expected to be:\n$content, but actually is:\n$actualContent\nHTTP status was $actualStatusCode"
```
for pystrings, inline messages may look unusual. newlines separations can be useful.